crates/swc_ecma_transforms_base/src/rename/analyzer/scope.rs

```diff
@@ -202,7 +220,7 @@ impl Scope {
                 continue;
             }
 
-            if self.data.all.contains(left) {
+            if self.data.all.contains(left) || self.data.imports.contains(left) {
```

Comment:
I don't understand why `all` didn't catch the reuse already.

Reply:
```rust
child.data.imports.extend(self.data.imports.iter().cloned());
self.data.all.extend(child.data.all.iter().cloned());
```

I added some descriptions to the fields